content/blog/2023/04/10/2023-04-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: "Jenkins March 2023 Newsletter"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- alyssat
+- dduportal
+- kmartens27
+- markewaite
+- kguerroudj
+- gounthar
+opengraph:
+  image: /images/post-images/2023/02/07/2023-02-07-jenkins-newsletter/centered-newsletter.png
+links:
+discourse: true
+---
+
+image:/images/post-images/2023/02/07/2023-02-07-jenkins-newsletter/centered-newsletter.png[Jenkins March Newsletter]
+
+==  Highlights
+
+* PowerPC docker images are back.
+* The Jenkins board, members of the Jenkins security team, and several others were involved in resolving an incorrect claim against a repository of the Jenkins GitHub organization.
+* Jenkins 2.397 and 2.387.2 are both using new Linux repository signing keys.
+* The Pipeline graph view plugin continues to evolve and improve as a Pipeline visualization replacement for Blue Ocean.
+* The number of pull requests merged for jenkins.io crossed into triple digits (101).
+
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+Jenkins' installers for Debian, and Red Hat have all been signed with new code PGP private keys.
+See the Jenkins link:/blog/2023/03/27/repository-signing-keys-changing/[blog post] for more details.
+The Jenkins installer for Windows and the Jenkins war file have also been signed with a new code signing certificate issued by DigiCert.
+Thanks to the Continuous Delivery Foundation for their help with the new code signing certificate.
+
+The Chinese translation of the Jenkins documentation has been unmaintained for almost two years.
+Rather than risk continuing bug reports from Chinese users that are following outdated instructions, we’ve removed the link to the outdated Chinese site.
```

Comment:
```suggestion
Rather than risk continuing bug reports from Chinese users that are following outdated instructions, we've removed the link to the outdated Chinese site.
```
Nit, I would do a global replace from `’` to `'`